Issue #557

mlantz opened the pull request:
Covers the cancel button, and the appropriate error code.

Commit:
Issue #557

Covers the cancel button, and the appropriate error code.

dansilivestru commented:
Just took a look, got a bit of a UI issue in that the button wraps to the next line on smaller screens. Can you merge next into your branch and have a look. this is only really a problem with images that are bigger then the screen size. Thoughts on how to best fix this in an elegant way?

mlantz committed:
Merge branch 'next' of https://github.com/blackberry/Ripple-UI into cancel-button

mlantz committed:
Issue #557

These minor changes address the issue of the button wrapping to the
next row on small screens.
